tickets/src/components/content/EventContent.js

```diff
@@ -98,8 +106,14 @@ export class EventContent extends Component {
             <Row>
               <DetailsField>
                 <DisplayDate>{dateString}</DisplayDate>
-                <Description>{description}</Description>
+                <Description>
+                  {description.split('\n').map(line => {
```

Comment:
Should this be `\n\n`? In some ways I'd be inclined to replace the paragraph tags with `<br />` - right now a single carriage return would mean a new paragraph.